frontend/database/155_fixed_qualitynomination_logs.sql

```diff
@@ -0,0 +1,38 @@
+/* Arregalndo los problemas que esta en warning pero no tienen ninguna nominacion con estatus warning*/
+UPDATE
+    `Problems` AS `p`
+SET
+    `visibility` = 2
+WHERE
+    `visibility` = 1 AND
+    (
+        SELECT
+            COUNT(`qualitynomination_id`)
+		FROM
+			`QualityNominations`
+		WHERE
+			(`status`='warning') AND
+            `nomination`='demotion' AND
+			`problem_id`=`p`.`problem_id`
+    )=0
+;
+
+/*Need next line to disable safe update*/
+SET SQL_SAFE_UPDATES = 0;
+
+/* Arregalndo el user_id y el from_status*/
```

Comment:
nit: todo el código debería ir en inglés, incluyendo los comentarios.

```suggestion
-- Fixing user_id and from_status
```

Reply:
si listo